servers/firecone/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "firecone",
+    "name": "Firecone.eu",
+    "description": "survival events and more",
+    "addresses": [
+        "mc.firecone.eu"
```

Comment:
This should just be firecone.eu the sub domain is auto-detected by Lunar Client.